src/Workspaces/Remote/ServiceHub/Services/RemoteSymbolSearchUpdateEngine.cs

```diff
@@ -9,8 +9,7 @@
 
 namespace Microsoft.CodeAnalysis.Remote
 {
-    internal partial class RemoteSymbolSearchUpdateEngine :
-        ServiceHubServiceBase, IRemoteSymbolSearchUpdateEngine, ISymbolSearchLogService, ISymbolSearchProgressService
+    internal partial class RemoteSymbolSearchUpdateEngine : ServiceBase, IRemoteSymbolSearchUpdateEngine, ISymbolSearchLogService, ISymbolSearchProgressService
```

Comment:
this got worse :)

Reply:
It got shorter (`ServiceHubServiceBase` -> `ServiceBase`). You mean it's too long a line? I think it's fine.